classes/ThanksItemFactory.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Claromentis\ThankYou;
+
+class ThanksItemFactory
+{
+	/**
+	 * @return ThanksItem
+	 */
+	public function Create()
+	{
+		return new ThanksItem();
+	}
+}
```

Comment:
Is this really necessary? 😄 It's used in one place, in the plugin. I think it can be removed and the `new ThanksItem();` could be moved there.

Reply:
I replaced it's usage in the plugin with a get for the specified Thank You (it only used it to check for existance).

Fixed in 788bfac06728f8ef555c8b0abb90126b2ba9ad83

Reply:
That's much better 👍